CentralManagement/CentralManagementSetupWeb.md

```diff
@@ -7,47 +7,7 @@ This is the Central Management website that gives an API and a web layer to cent

```

Comment:
Go into your VS Code settings and add `"files.eol": "\n"`. If it is set to "auto", this is what you get in the TOC.

Reply:
Made VS Code setting changes to get TOC working correctly.